core/src/main/scala/org/apache/spark/h2o/H2OContext.scala

```diff
@@ -92,6 +93,10 @@ class H2OContext private(val sparkSession: SparkSession, conf: H2OConf) extends
     * otherwise it creates new H2O cluster living in Spark
     */
   def init(): H2OContext = {
+    ShutdownHookManager.addShutdownHook(10)(() => {
```

Comment:
Would it be possible to add some comment explaining priority `10`?

Reply:
Sure, I need to provide comment in the code. We need to have higher ( lower number) priority then spark shutdown hooks so we stop H2O first and then let Spark stop itself

Reply:
I will make a constant for it with explanation